third_party/move/move-core/types/Cargo.toml

```diff
@@ -14,6 +14,7 @@ anyhow = { workspace = true }
 arbitrary = { workspace = true, features = ["derive_arbitrary"], optional = true }
 bcs = { workspace = true }
 bytes = { workspace = true }
+dearbitrary = { version = "1.0.4", optional = true, features = ["derive"] }
```

Comment:
can we use the same pattern where the external dependency version is declared at the workspace level?